modules/Standardization/src/client/Standardization.html

```diff
@@ -0,0 +1,264 @@
+            <th style="width: 100px;">Standardization Reason</th>
+        </tr>
+        </thead>
+        <tbody style="">
+
+        </tbody>
+    </table>
+
+    <div class="modal bv_standardizationHistoryModal hide">
+        <div class="modal-header">
+            <h3>Standardization History Table Attributes</h3>
+        </div>
+        <div class="modal-body">
+            <div>
+                <ul>
+                    <li># Structures Standardized: Total number of structures run through the standardizer.</li>
+                    <li># Structures Updated: Count of parent structures that were changed as a result of standardization.</li>
+                    <li># Structures Changed: Count of structures that are no longer the same as their parent structure taking into account tautomers.</li>
+                    <li># Display Change: Count of structures that are no longer exact string matches for their parent structure.</li>
+                    <li># New Duplicates: Count of structures that are now the same as one or more parent structures taking into account tautomers, stereo comments and stereo categories.</li>
+                    <li># As Drawn Display Change: Count of structures that are no longer exact string matches for their originally drawn as structure.</li>
+                    <li># Existing Duplicates: Count of structures that were already the same as one or more parent structures taking into account tautomers, stereo comments and stereo categories.</li>
+                    <li>Settings Hash: Hash of the effective standardization settings.</li>
+                </ul>
+            </div>
+        </div>
+        <div class="modal-footer">
+            <button type="button" class="btn bv_standardizationHistoryModalCloseBtn">Close</button>
+        </div>
+    </div>
+
+</script>
+
+<script type="text/template" id="StandardizationDryRunReportStatsView" xmlns="http://www.w3.org/1999/html">
+    <h4>Latest Standardization Dry Run Results
+        <a class="bv_dryRunResultsInfo">
+            <i class="icon-question-sign"></i>
+        </a>
+    </h4>
+    <table class="table table-bordered table-striped table-hover bv_standardizationDryRunReportStatsTable" style="margin: 0px; font-size:70%; ">
+    </table>
+    <div class="alert bv_getDryRunReportStatsError">
+        There was an error getting the summary of the latest standardization dry run results. Please contact an administrator.
+    </div>
+
+    <div class="modal bv_dryRunResultsModal hide">
+        <div class="modal-header">
+            <h3>Dry Run Results Table Attributes</h3>
+        </div>
+        <div class="modal-body">
+            <div>
+                <ul>
+                    <li># Structures Standardized: Total number of structures run through the standardizer.</li>
+                    <li># Structures Updated: Count of parent structures that were changed as a result of standardization.</li>
```

Comment:
We should be more specific with these messages since these are complex concepts and very close meanings.
I think this one is "Number of structures whose MOL representation changed at all as a result of standardization"